app/assets/javascripts/services/builds_svc.js

```diff
@@ -0,0 +1,22 @@
+samson.service('buildsService', function($http, $q, httpErrorService) {
+  var config = {
+    headers: {
+      'Accept': 'application/json'
+    }
+  };
```

Comment:
Yep, I realised that in the meantime. I usually like to have the backend deciding which format to use depending on the 'Accept' header sent by the client in the HTTP request. This could even be set globally, but I was not sure about the impact, so I'm doing it service by service. :)

But I'm fine both ways as well. This type of "convention over configuration" can be found in many other frameworks as well.